### DISCUSSION TITLE
Is EV4 possible?

### DISCUSSION THREAD

**Discussion by RMNB** [srozum/film_camera_tester#52] [Q&A] [answered]
Hello! 
Have a camera I'm trying to adjust and according to its service manual, I need EV4 and EV14. I noticed that this project used to say EV 4..16, but at some point it was updated to EV5. Is EV4 practicable with this hardware?

**Comment by srozum** [comment]
Hello,

yes EV4 light level is possible to achieve with this light source. You are correct, early versions had range starting from ev4 but in later versions I had to change it. 

although it’s possible it’s not practical. Even at ev5 ambient light bleeds through the diffuser and affects readings, unless you in a dark room or somehow block ambient light.

**Reply by RMNB** [reply]
Thanks. I haven't built the device but if I understand correctly, EV4 is no longer in the menu. Would it be possible to send me the settings the firmware uses to generate the EV levels so I could recrate EV4-14?

**Comment by ic-racer** [comment, marked as answer]
You could also use a 1 stop ND filter. Rosco 0.3ND is about $8 usd.
